src/main/java/org/fineract/messagegateway/sms/service/EmailService.java

```diff
@@ -0,0 +1,95 @@
+package org.fineract.messagegateway.sms.service;
+
+import org.fineract.messagegateway.MessageGateway;
+import org.fineract.messagegateway.sms.data.EmailRequestDTO;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.client.ClientHttpRequestFactory;
+import org.springframework.http.client.SimpleClientHttpRequestFactory;
+import org.springframework.mail.SimpleMailMessage;
+import org.springframework.mail.javamail.JavaMailSender;
+import org.springframework.mail.javamail.JavaMailSenderImpl;
+import org.springframework.stereotype.Service;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.web.client.RestTemplate;
+
+import java.util.Properties;
+
+
+@Service
+public class EmailService {
+
+    private static final Logger logger = LoggerFactory.getLogger(EmailService.class);
+    @Autowired
+    private JavaMailSender mailSender;
+
+    @Value("${spring.mail.host}")
```

Comment:
why do we have to specify 'spring' here? looks like an overhead. we are not using this anywhere else.